app/src/main/java/edu/temple/templetag/LoginActivity.java

```diff
@@ -84,8 +84,6 @@ private void startHomeActivity() {
 
     private void startSignUpActivity() {
         signUpIntent = new Intent(LoginActivity.this, SignUpActivity.class);
-        signUpIntent.addFlags(Intent.FLAG_ACTIVITY_CLEAR_TASK | Intent.FLAG_ACTIVITY_NEW_TASK);
```

Comment:
Why get rid of these flags?

Reply:
Because we still want the user to still be able to go back to login page from signup page